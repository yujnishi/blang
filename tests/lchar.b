src "hello, world";
dst "xxxxxxxxxxxxxxxxxxxx";

main() {
    auto c,i;

    printf("dst = '%s'*n",dst);
    i = 0;
    while ( (c=char(src,i)) != '*e' ) lchar(dst,i++,c);
    lchar(dst,i,'*e');
    printf("dst = '%s'*n",dst);
}

