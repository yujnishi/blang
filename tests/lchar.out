dst = 'xxxxxxxxxxxxxxxxxxxx'
dst = 'hello, world'
